### PR TITLE
Aurora 2: The 3/4thening

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
Totally overhauls the server look! This was a PR lovingly created by the entire development team.

Its development begins around December, when we began talking about how to create an Aurora-specific style. For a long while, we'd forced new sprites to be 3/4ths, but the walls, windows and airlocks were particularly jarring in respect to that. Furthermore, we couldn't just port new airlocks or walls - we wanted our own look, created by our own spriters. We wanted a unique server identity, instead of a mishmash of sprites from different servers.

And so, here it is: the fruit of over 8 months of work! By MattAtlas, FluffyGhost, Dreamy (Code, Mapping); Atteria, Wezzy, Nauticall, Kyres1 (Sprites, Mapping, Code), Gecko (Sprites).

```
A thing of beauty, I know
Will never fade away
What you did to me, I know
Said what you had to say
But a thing of beauty
Will never fade away
```

Examples:
dreamseeker_np8lKPYG6o
dreamseeker_WZjOMYGixM
dreamseeker_Ocf4dkzpFZ
dreamseeker_AmeFSJV1z2
dreamseeker_D1mCMzylNM

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!wip

File: code/game/machinery/doors/airlock.dm [hunk 1]
```diff
@@ -780,99 +939,183 @@ About the new airlock wires panel:
...
 		sleep(10)
-		hasShocked = FALSE
+		has_shocked = FALSE
 		return 1
 	else
 		return 0
 
 // Only set_light() if there's a change, no need to waste processor cycles with lighting updates.
-/obj/machinery/door/airlock/update_icon()
+/obj/machinery/door/airlock/update_icon(var/state = 0)
```
Comment (Generalcamo) [inline code/game/machinery/doors/airlock.dm:953]:
```suggestion
/obj/machinery/door/airlock/update_icon(state = 0)
```

**Comment by Eyeveri** [comment]
This is beautiful. 

Will this change anything about third party ships?

File: code/game/objects/structures/stool_bed_chair_nest/sofa.dm [hunk 2]
```diff
@@ -0,0 +1,225 @@
+/obj/structure/bed/stool/chair/sofa
+	name = "sofa"
+	desc = "An old design, but it still does the job of being a sofa."
+	icon = 'icons/obj/structure/sofa.dmi'
+	icon_state = "sofa_middle"
+	base_icon = "sofa_middle"
+
+/obj/structure/bed/stool/chair/sofa/brown/New(var/newloc)
```
Comment (Generalcamo) [inline code/game/objects/structures/stool_bed_chair_nest/sofa.dm:8]:
```suggestion
/obj/structure/bed/stool/chair/sofa/brown/New(newloc)
```
The entire rest of this file too

File: code/game/objects/structures/window.dm [hunk 3]
```diff
@@ -568,19 +572,31 @@
...
+	can_blend_with = list(
+		/turf/simulated/wall/shuttle/scc_space_ship,
+		/obj/structure/window/shuttle/scc_space_ship
+	)
+
 
 /obj/structure/window/shuttle/scc_space_ship/cardinal
 	smooth = SMOOTH_MORE
 
+/obj/structure/window/shuttle/scc_space_ship/cardinal_smooth(adjacencies, var/list/dir_mods)
```
Comment (Generalcamo) [inline code/game/objects/structures/window.dm:596]:
```suggestion
/obj/structure/window/shuttle/scc_space_ship/cardinal_smooth(adjacencies, list/dir_mods)
```

**Review by Generalcamo — COMMENTED** [review]
Sitting down and reviewing this is one of my goals. This is an initial passthrough review, will be more through in the future

File: code/_helpers/icon_smoothing.dm [hunk 4]
```diff
@@ -230,6 +238,65 @@
 
 		underlays = U
 
+/turf/proc/get_underlays(var/list/adjacencies)
```
[the remark this answers is not in the document]
Comment (NonQueueingMatt) [reply code/_helpers/icon_smoothing.dm:241]: This is not a standard we use consistently

**Comment by Greenjoe12345** [comment]
ledges have an issue wiht passing them
image
I cannot move to that direction from the tile with the ledge on it, but can move up the ledge without doing anything.
same in the other direction approaching the ledge too.

**Review by Arrow768 — APPROVED** [review]

**Comment by Greenjoe12345** [comment]
image
this platform should be over characters so it looks like they're on the lower level

**Comment by Greenjoe12345** [comment]
image
same for wallmounted stuff?

**Comment by Greenjoe12345** [comment]
image
chair back layering

**Review by DreamySkrell — APPROVED** [review]

**Comment by Yakunan** [comment]
Wow. Simply beautifull it looks so clean. Great job everyone!

**Comment by Greenjoe12345** [comment]
External airlocks shock you.

**Pull request merged**